src/components/Globals/ConfigMetric.ts

```diff
@@ -0,0 +1,54 @@
+import styled from 'styled-components/native';
+import Colors from '../../constants/Colors';
+import FontSize from '../../constants/FontSize';
+
+export const ConfigMetricWrapper = styled.View<{
+  borderBottom?: string;
+}>`
+  flex-direction: row;
+  padding-top: 20px;
+  padding-bottom: 20px;
+  padding-left: 10px;
+  padding-right: 10px;
+  align-items: center;
+  border-bottom-width: 1px;
+  border-bottom-color: ${({ borderBottom }) => borderBottom || Colors.borders};
+  justify-content: center;
+`;
+
+export const ConfigLableContainer = styled.View`
```

Comment:
there is a typo: it should be `Label` instead of `Lable`. Remember to update everywhere when fixing